releases.json

```diff
@@ -1530,6 +1530,15 @@
       "1.9-1"
     ]
   },
+  "iceberg-cpp": {
+    "dependency_names": [
+      "iceberg",
+      "iceberg_rest",
```

Comment:
Upstream it looks like this mistakenly calls `meson.override_dependency('iceberg-rest', ...)` so need to fix that before release